### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by rex4539**

### CODE REVIEW

File: packages/blockchain/test/clique.spec.ts [hunk 1]
```diff
@@ -583,7 +583,7 @@ tape('Clique: Initialization', (t) => {
     "Clique Voting: Ensure that pending votes don't survive authorization status changes",
     async (st) => {
       // This corner case can only appear if a signer is quickly added, removed
-      // and then readded (or the inverse), while one of the original voters
+      // and then read (or the inverse), while one of the original voters
```
Comment (acolytec3) [inline packages/blockchain/test/clique.spec.ts:586]: The original wording is correct here. You could change to `re-added`

**Review by acolytec3 — DISMISSED** [review]
Thanks for the PR. One correction and then we can merge.

**Comment by rex4539** [comment]
Rebased to fix conflicts

**Review by holgerd77 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**